.devcontainer/Dockerfile

```diff
@@ -3,44 +3,45 @@
 
 # See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.195.0/containers/cpp/.devcontainer/base.Dockerfile
 # [Choice] Debian / Ubuntu version (use Debian 11/9, Ubuntu 18.04/21.04 on local arm64/Apple Silicon): debian-11, debian-10, debian-9, ubuntu-21.04, ubuntu-20.04, ubuntu-18.04
-ARG VARIANT=ubuntu-20.04
-FROM mcr.microsoft.com/vscode/devcontainers/cpp:0-${VARIANT}
+ARG VARIANT=debian-12
```

Comment:
Not sure whether it is a good practice to change ubuntu to debian. Some developers may like to use ubuntu? Could we add another file for ubuntu, e.g., Dockerfile.ubuntu?

Reply:
OK, got it, thanks.